draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -4918,21 +4918,20 @@ Host: server.example.com
    into the protocol's wire format.
 </t>
 <t>
-   CONNECT is intended only for use in requests to a proxy.
+   CONNECT is intended for use in requests to a proxy.
+   The recipient can establish a tunnel either by directly connecting to
+   the server identified by the request target or, if configured to use
+   another proxy, by forwarding the CONNECT request to the next inbound proxy.
    An origin server &MAY; accept a CONNECT request, but most origin servers
    do not implement CONNECT.
 </t>
 <t>
-   The recipient proxy can establish a tunnel either by directly connecting to
-   the request target or, if configured to use another proxy, by forwarding
-   the CONNECT request to the next inbound proxy.
    Any <x:ref>2xx (Successful)</x:ref> response indicates
    that the sender (and all inbound proxies) will switch to tunnel mode
-   immediately after the blank line that concludes the successful response's
-   header section; data received after that blank line is from the server
-   identified by the request target.
+   immediately after the response header section; data received after that
+   header section is from the server identified by the request target.
```

Comment:
I think that this is OK.  HTTP/2 and /3 have subtly different treatment here in that they only forward DATA frames to the tunnel, whereas HTTP/1.1 has no similar means of distinguishing tunnel data from other stuff.  I think that's OK at this level though.